docs/examples/Makefile

```diff
@@ -17,3 +17,13 @@ compile:
 
 # TODO: type check the different files with the right checker;
 #   some tests expect errors, compare against expected errors.
+
+clean:
+	rm -f *.class
+	cd fenum-extension && $(MAKE) clean
```

Comment:
I like the style in checker/tests/nullness-extra/Makefile better.
Is there a reason for the different approach here?

Reply:
I am following the style for `all` target. I have uniform it with checker/tests/nullness-extra/Makefile now.